waspc/e2e-test/Util.hs

```diff
@@ -0,0 +1,82 @@
+module Util
+  ( newCommand,
+    newAndCompileCommand,
+    newAndCompileAndMigrateCommand,
+    runGoldenTest,
+  )
+where
+
+import Control.Monad (filterM)
+import Data.List (intercalate, sort)
+import Data.Text (pack, replace, unpack)
+import System.Directory (doesFileExist)
+import System.Directory.Recursive (getDirFiltered)
```

Comment:
What about listDirectoryDeep that we have in Wasp.Util.IO -> would that maybe be a better choice than using `getDirFiltered`? I am not sure, just mentioning it in case you are not aware that we have that function.

Reply:
Yeah, I remembered that one. I had not moved to SP yet so `getDirFiltered` was nice. It also supported filters, which `listDirectoryDeep` does not appear to. I could filter after, but going into `node_modules` unnecessarily could be an expensive proposition.

Reply:
Ah uff, ok that is very important indeed -> if `getDirFiltered` does filtering while searching, which it does, that is very important due to `node_modules` as you mentioned. Makes sense then, certainly stick with it!